dojo/jira_link/views.py

```diff
@@ -218,6 +135,109 @@ def webhook(request, secret=None):
     return HttpResponse('')
 
 
+def check_for_and_create_comment(parsed_json):
+    """
+        example incoming requests from JIRA Server 8.14.0
+        {
+        "timestamp":1610269967824,
+        "webhookEvent":"comment_created",
+        "comment":{
+            "self":"https://jira.host.com/rest/api/2/issue/115254/comment/466578",
+            "id":"466578",
+            "author":{
+                "self":"https://jira.host.com/rest/api/2/user?username=defect.dojo",
+                "name":"defect.dojo",
+                "key":"defect.dojo", # seems to be only present on JIRA Server, not on Cloud
+                "avatarUrls":{
+                    "48x48":"https://www.gravatar.com/avatar/9637bfb970eff6176357df615f548f1c?d=mm&s=48",
+                    "24x24":"https://www.gravatar.com/avatar/9637bfb970eff6176357df615f548f1c?d=mm&s=24",
+                    "16x16":"https://www.gravatar.com/avatar9637bfb970eff6176357df615f548f1c?d=mm&s=16",
+                    "32x32":"https://www.gravatar.com/avatar/9637bfb970eff6176357df615f548f1c?d=mm&s=32"
+                },
+                "displayName":"Defect Dojo",
+                "active":true,
+                "timeZone":"Europe/Amsterdam"
+            },
+            "body":"(Valentijn Scholten):test4",
+            "updateAuthor":{
+                "self":"https://jira.host.com/rest/api/2/user?username=defect.dojo",
+                "name":"defect.dojo",
+                "key":"defect.dojo",
+                "avatarUrls":{
+                    "48x48":"https://www.gravatar.com/avatar/9637bfb970eff6176357df615f548f1c?d=mm&s=48",
+                    "24x24""https://www.gravatar.com/avatar/9637bfb970eff6176357df615f548f1c?d=mm&s=24",
+                    "16x16":"https://www.gravatar.com/avatar/9637bfb970eff6176357df615f548f1c?d=mm&s=16",
+                    "32x32":"https://www.gravatar.com/avatar/9637bfb970eff6176357df615f548f1c?d=mm&s=32"
+                },
+                "displayName":"Defect Dojo",
+                "active":true,
+                "timeZone":"Europe/Amsterdam"
+            },
+            "created":"2021-01-10T10:12:47.824+0100",
+            "updated":"2021-01-10T10:12:47.824+0100"
+        }
+        }
+    """
+    if (comment := parsed_json.get("comment", None)) is not None:
```

Comment:
Flipping this logic would let you eliminate a level of nesting throughout this block

```suggestion
    comment = parsed_json.get("comment", None)
    if comment is None:
        return
    ....
```

Reply:
Rip I didn't think about the rest of the function needed to be unindented now